README.md

```diff
@@ -66,12 +66,6 @@
 
```

Comment:
**[OPTIONAL]**

- [ ] It would be great if you add your own logo (Not Microverse logo) at the top of your readme file to make yourself a brand in the tech industry to catch the eyes of recruiters 👍